pywinauto/unittests/test_application.py

```diff
@@ -46,6 +46,9 @@
 import ctypes
 import mock
 
+from pywinauto.controls.common_controls import TrackbarWrapper

```

Comment:
Both new imports should be moved below `sys.path.append('.')` 
Helps to run individual tests from command line:
```
python pywinauto\unittests\test_application.py DesktopWin32WindowSpecificationTests.test_non_magic_desktop
```